src/main/java/com/atlassian/db/replica/spi/DatabaseCluster.java

```diff
@@ -0,0 +1,10 @@
+package com.atlassian.db.replica.spi;
+
+import com.atlassian.db.replica.api.Database;
+
+import java.sql.SQLException;
+import java.util.stream.Stream;
+
+public interface DatabaseCluster {
```

Comment:
This is the most important part of the pull-request (API). @dwatl Do you think it will expose enough information to build replica-discovery?

Reply:
How do we plan to implement Database? We need cluster URL and replica id

Reply:
Cluster URL seems to be a constant for the cluster, and replica id can be fetched from the database. Am I wrong? Can you elaborate?

Reply:
Hmm, correct!